api/artifact_lister.go

```diff
@@ -77,6 +77,7 @@ func (lister ArtifactLister) ListIntermediateFileDetails(appSlug string, buildSl
 	for i := 1; i <= len(buildSlugs); i++ {
 		res := <-listResults
 		if res.err != nil {
+			lister.logger.Warnf("Failed to list artifacts for build %s: %w", res.buildSlug, res.err)
```

Comment:
`%w` should be `%s`
https://github.com/bitrise-steplib/steps-fastlane/pull/90#discussion_r1026463564